src/components/AmountFilter.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { Button, Popover, InputLabel, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import RemoveIcon from '@material-ui/icons/Remove';
+import AmountTextField from './AmountTextField';
+
+const useStyles = makeStyles((theme) => ({
+  buttonStyle: {
+    margin: theme.spacing(1),
+    maxWidth: 120,
```

Comment:
Why is this necessary?

Reply:
I was trying to make the width somewhat similar to the other filters but it looks consistent without it so I took it off.